Update .NET library docs (#10759)
PostHog/posthog.com: contents/docs/feature-flags/snippets/groups-flags-dotnet.mdx

> To check a flag that's matching by companies, specify groups in relevant call:

Reviewer: ```suggestion
To check a flag matching a "company" group, specify the group in relevant call:
```
